core/templates/pages/about-page/about-page.component.css

```diff
@@ -281,6 +281,17 @@
     height: 50vh;
   }
 }
+@media (orientation: landscape) and (max-width: 960px) {
```

Comment:
I don't really see a point of adding this media query

Reply:
Actually, yes! I tried to add more clarity, but yeah, you are right. We can do that as well. Updated.